specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,2059 @@
+                "description": "Gets a custom API by name for a specific subscription and resource group",
+                "x-ms-examples": {
+                    "Get a custom API": {
+                        "$ref": "./examples/customApisGet.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/resourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/exportParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "A custom API",
+                        "schema": {
+                            "$ref": "#/definitions/CustomApiDefinition"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Custom APIs"
+                ],
+                "summary": "Replaces an existing custom API",
+                "operationId": "CustomApis_CreateOrUpdate",
+                "description": "Creates or updates an existing custom API",
+                "x-ms-examples": {
+                    "Replace a custom API": {
+                        "$ref": "./examples/customApisCreateOrUpdate.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/resourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/customApiParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The replacing custom API",
+                        "schema": {
+                            "$ref": "#/definitions/CustomApiDefinition"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": [
+                    "Custom APIs"
+                ],
+                "summary": "Update an existing custom API",
+                "operationId": "CustomApis_Update",
+                "description": "Update an existing custom API's tags",
```

Comment:
"Updates an ..." to be consistent with other API operation summaries.

Reply:
Changed